exaudfclient/docs/script_options_requirments.md

```diff
@@ -0,0 +1,330 @@
+The value should be handled according to the [Java specification for identifies](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.8).
+
+Needs: dsn
+
+Tags: V1
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+
+### Java %scriptclass Option Handling V2
+`req~java-scriptclass-option-handling-v2~1`
+
+The Java parser handler must correctly identify the first `%scriptclass` option and remove any additional occurrences of this option within the script code.
+The value should be handled according to the [Java specification for identifies](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.8).
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %jar Option Handling V1
+`req~java-jar-option-handling-v1~1`
+
+The Java parser handler must find multiple %jar options. The values are to be interpreted as the Java CLASSPATH: `<file1>:<file2>:...:<filen>`.
+The Java parser handler shall split the entries by the colon character. The Java parser handler shall identify duplicated files and order the result of all `%jar` options alphabetically.
+
+Needs: dsn
+
+Tags: V1
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %jar Option Handling V2
+`req~java-jar-option-handling-v2~1`
+
+The Java parser handler must find multiple %jar options. The values are to be interpreted as the Java CLASSPATH: `<file1>:<file2>:...:<filen>`.
+The Java parser handler shall split the entries by the colon character. The Java parser handler must keep duplicates. The order of the entries must not change.
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %jar Option Trailing White Space Handling
+`req~java-jar-option-trailing-white-space-handling~1`
+
+The Java parser handler must remove trailing white spaces for `%jar` option values if they are part of the escape sequence '\ '. Escape sequences at the end of a found `%jar` option of the form `\ ` must be replaced with ' '.
+This approach provides backwards compatibility for most existing UDF's from customers.
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+Depends: 
+- `req~white-spaces-script-options-parsing-v2~1`
+
+### Java %jvmoption Handling
+`req~java-jvmoption-handling~1`
+
+The Java parser handler must find multiple %jvmoption options, allowing duplicates and maintaining order.
+
+Needs: dsn
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %import Option Replace Referenced Sripts
+`req~java-import-option-replace-referenced-scripts~1`
+
+For each found %import option, the Java parser handler must request and replace the referenced scripts recursively. This means,
+if the referenced scripts contain also `%import` options, the implementation must replace those, too.
```

Comment:
Ignore already imported scripts is missing as a requirement

Reply:
Done. Added in `req~java-import-option-handling~1`